pages.pt_PT/common/yt-dlp.md

```diff
@@ -0,0 +1,37 @@
+# yt-dlp
+
+> Um fork do youtube-dl com funcionalidades e correções adicionais.
+> Descarrega vídeos do YouTube e de outros websites.
+> Mais informações: <https://github.com/yt-dlp/yt-dlp>.
+
+- Descarregar um vídeo ou playlist (com as predefinições do comando abaixo):
```

Comment:
Eu já vi "lista de reprodução" e parece-me uma tradução aceitável. Preferes mesmo "playlist"?

Reply:
Outra coisa: acho que fica estranho "com as predefinições do comando abaixo". Se calhar eu poria algo como "com as opções predefinidas". O que achas?

Reply:
> Eu já vi "lista de reprodução" e parece-me uma tradução aceitável. Preferes mesmo "playlist"?

Também ponderei. O YouTube em si não faz a tradução de "playlist" para "lista de reprodução", portanto optei pelo estrangeirismo, é mais conciso e utilizado no geral, na minha opinião.

> Outra coisa: acho que fica estranho "com as predefinições do comando abaixo". Se calhar eu poria algo como "com as opções predefinidas". O que achas?

Concordo, vou mudar. 🙂

Reply:
> O YouTube em si não faz a tradução de "playlist" para "lista de reprodução", portanto optei pelo estrangeirismo, é mais conciso e utilizado no geral, na minha opinião.

Ok, sounds good to me! :+1: